docs/development.md

```diff
@@ -227,7 +227,30 @@ experimental-features = nix-command flakes
 
 ## Quick Usage Guide
 
-Get a Nix development shell:
+### With `direnv`
+
+The recommended way to develop is to install [`direnv`](https://direnv.net/)
```

Comment:
Who recommends and why?

I think it says below "From then on, every time you `cd` in that same folder ...". 

Would be good to move this forward, as it is easier to follow along with a goal in mind.

Reply:
Not sure what answer you expect. "Tiziano / The Oak team recommends `direnv` because $REASON ?" 

Anyways, I moved the cd part earlier, and removed the recommendation sentence (even though I think it was fine personally)

Reply:
> Tiziano /The Oak team recommends 

Yes, something along, e.g., "In Oak, we use ...". 

I literally could not tell whether this is an official Nix recommendation or some other power.